notebooks/week2/05_log_and_register_fe_model.py

```diff
@@ -0,0 +1,185 @@
+# Databricks notebook source
+# MAGIC %pip install ../power_consumption-0.0.1-py3-none-any.whl
```

Comment:
your use case is taxi, right?

Reply:
i commented it out. but yes. i fixed it anyway

Reply:
it was a comment but i changed this